.github/ISSUE_TEMPLATE/0_bug.yml

```diff
@@ -0,0 +1,76 @@
+name: Bug Report
+description: Create a bug report.
+labels:
+  - type::bug
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Processing bug reports is time-consuming so please fill out the following form as completely as possible. **The more complete this form is, the more likely we can provide a speedier response.**
+
+        > ⚠️ Reports that are incomplete or missing information will simply be closed.
+
+        Since there are already a lot of open issues, please take a moment to search existing ones to see if your bug has already been reported. If you find something related, please upvote that report and provide additional details as necessary.
+
+        > 💐 Thanks for helping to make Conda better! We would be unable to do the things we do without our community!
```

Comment:
We did a lot of word-smithing with the old form to make sure tone isn't as direct and highlights the reasons to ask these questions. I'll update it a bit to fit with the conda bug report form, especially not promising to have a speedier response, that's gonna backfire ;)

```suggestion
        Because processing new bug reports is time-consuming, we would like to ask you to fill out the following form to the best of your ability and as completely as possible.
        
        > **Note**
        > ⚠️ Bug reports that are incomplete or missing information may be closed as inactionable.

        Since there are already a lot of open issues, please also take a moment to search existing ones to see if your bug has already been reported. If you find something related, please upvote that issue and provide additional details as necessary.

        💐 Thank you for helping to make Conda better. We would be unable to improve Conda without our community!
```

Reply:
I agree with word-smithing for a softer tone, just want to make sure we keep the text as short as possible bc the longer the text the less likely it's read.

Reply:
I like this suggestion. And Jannis's edits are a bit shorter. Win/win!